lib/hot/widget.lua

```diff
@@ -0,0 +1,327 @@
+---------------------------------------------------------------------------
+--- Widget which show current hotkeys.
+--
+-- @author Yauheni Kirylau &lt;yawghen@gmail.com&gt;
+-- @copyright 2014-2015 Yauheni Kirylau
+-- @release @AWESOME_VERSION@
+-- @module hot
+---------------------------------------------------------------------------
+
+local capi = {
+  screen = screen,
+  client = client,
+  keygrabber = keygrabber,
+}
+local awful = require("awful")
+local common = require("awful.widget.common")
+local wibox = require("wibox")
+local beautiful = require("beautiful")
+local dpi = beautiful.xresources.apply_dpi
+
+
+local markup = {}
```

Comment:
btw, what do you think about copying whole `lain.util.markup`[1] into awesome?

[1] https://github.com/copycat-killer/lain/blob/master/util/markup.lua